### PR TITLE
Register ACF Blocks

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
### Description

Adds block for `Key Indicators`, `Key Statistics`, `Explore` & `Banners`

What did you build? Give some context...

### Screenshots

Add screenshots of your feature.

### Steps To Verify Feature

How will your Lead Engineer and/or stakeholder test this?

1.
2.
3.

### CODE REVIEW

File: themes/wds_headless/inc/acf-pro.php [hunk 1]
```diff
@@ -67,6 +67,74 @@ function wds_acf_blocks_init() {
...
+		);
+
+		// Explore block.
+		acf_register_block_type(
+			[
+				'name'            => 'explore-list',
+				'title'           => esc_html__( 'Explore List', 'wds' ),
+				'description'     => esc_html__( 'A block to display list of explore cards.', 'wds' ),
+				'render_callback' => '',
+				'category'        => 'wds-content',
```
Comment (KobbyMmo) [inline themes/wds_headless/inc/acf-pro.php]: can we customize the category?

File: themes/wds_headless/inc/acf-pro.php [hunk 2]
```diff
@@ -67,6 +67,74 @@ function wds_acf_blocks_init() {
...
+				'align'           => 'wide',
+				'supports'        => $supports,
+			]
+		);
+
+		// Explore block.
+		acf_register_block_type(
+			[
+				'name'            => 'explore-list',
+				'title'           => esc_html__( 'Explore List', 'wds' ),
```
Comment (KobbyMmo) [inline themes/wds_headless/inc/acf-pro.php:92]: not sure if we need the 'wds' [here](https://developer.wordpress.org/reference/functions/esc_html__/)

File: themes/wds_headless/inc/acf-pro.php [hunk 3]
```diff
@@ -67,6 +67,74 @@ function wds_acf_blocks_init() {
...
+		);
+
+		// Key Indicators Block.
+		acf_register_block_type(
+			[
+				'name'            => 'key-indicators-list',
+				'title'           => esc_html__( 'Key Indicators List', 'wds' ),
+				'description'     => esc_html__( 'A block to display list of key Indicators.', 'wds' ),
+				'render_callback' => '',
+				'category'        => 'wds-content',
```
Comment (KobbyMmo) [inline themes/wds_headless/inc/acf-pro.php]: same as above

File: themes/wds_headless/inc/acf-pro.php [hunk 4]
```diff
@@ -67,6 +67,74 @@ function wds_acf_blocks_init() {
...
+		);
+
+		// Key Statistics block.
+		acf_register_block_type(
+			[
+				'name'            => 'key-statistics-list',
+				'title'           => esc_html__( 'Key Statics List', 'wds' ),
+				'description'     => esc_html__( 'A block to display list of key statistics', 'wds' ),
+				'render_callback' => '',
+				'category'        => 'wds-content',
```
Comment (KobbyMmo) [inline themes/wds_headless/inc/acf-pro.php]: and here too

**Review by KobbyMmo — APPROVED** [review]
looks good to me

File: themes/wds_headless/inc/acf-pro.php (hunk 1 above)
Comment (KhadijaMahanga) [reply themes/wds_headless/inc/acf-pro.php]: @KobbyMmo here, we can pass any string. where this block will be grouped to in blocks list? So its possible.

File: themes/wds_headless/inc/acf-pro.php (hunk 2 above)
Comment (KhadijaMahanga) [reply themes/wds_headless/inc/acf-pro.php:92]: @KobbyMmo, that's just the theme name string. It is not a required parameter...but having `wds` is better than having `default` (a string that will be added if you don't add anything)

File: themes/wds_headless/inc/acf-pro.php (hunk 1 above)
Comment (KhadijaMahanga) [reply themes/wds_headless/inc/acf-pro.php]: I have replaced that with `Section`

**Pull request merged**